app/controllers/discovered_hosts_controller.rb

```diff
@@ -202,16 +202,12 @@ def controller_permission
 
   def action_permission
     case params[:action]
-      when 'refresh_facts', 'reboot', 'reboot_all'
-        :view
-      when 'update_multiple_location', 'select_multiple_organization', 'update_multiple_organization', 'select_multiple_location'
+      when 'refresh_facts', 'reboot', 'reboot_all', 'update_multiple_location', 'select_multiple_organization', 'update_multiple_organization', 'select_multiple_location'
```

Comment:
Since all the actions reachable via multiple selections are edit-like, I'd leave it as is. The same case is Foreman Core Hosts controller.